guides/release/contributing/repositories.md

```diff
@@ -61,3 +62,6 @@ These libraries are part of the Ember asset output, but development of them take
 
 * **rsvp.js** - Implementation of the of Promises/A+ spec used by Ember.
 * [https://github.com/tildeio/rsvp.js](https://github.com/tildeio/rsvp.js)
+
+## Many different ways to contribute
+There are many different ways to contribute to Ember in which you’ll be able to learn more about the project along the way: For example, you can [write and share an Ember addon with the community](https://ember-cli.com/extending/#developing-addons-and-blueprints) or collaborate with the authors of your favorite addon to make it even better. You can share any of your experiences with Ember by blogging, screencasting or speaking about them at your [local Ember meetup group](https://www.emberjs.com/community/meetups/) or organizing local Ember events on your own. Helping to make Ember even better understood is in itself a great way to contribute to the community.
```

Comment:
```suggestion
There are many different ways to contribute to Ember in which you'll be able to learn more about the project along the way: For example, you can [write and share an Ember addon with the community](https://ember-cli.com/extending/#developing-addons-and-blueprints) or collaborate with the authors of your favorite addon to make it even better. You can share any of your experiences with Ember by blogging, screencasting or speaking about them at your [local Ember meetup group](https://www.emberjs.com/community/meetups/) or organizing local Ember events on your own. Helping to make Ember even better understood is in itself a great way to contribute to the community.
```

I'm newer to this repo. The linting is super sensitive re: https://travis-ci.org/github/ember-learn/guides-source/builds/676488647#L587

//

As for the second error https://travis-ci.org/github/ember-learn/guides-source/builds/676488647#L588, I think we need to add the word `screencasting` to https://github.com/ember-learn/guides-source/blob/master/.local.dic 

Ref: old commit https://github.com/ember-learn/guides-source/commit/7526a97f843ce0bead52e6813d6c866fd4de355e#diff-ad770adfb2c0dcd210aa570bbf9fcb28

I need to take a closer look at all the markdown linting and set things up locally, hopefully this weekend. But if you want to try adding that as a commit now, we can prob get this merged in. Hope this helps @rajasegar

Reply:
@amyrlam Yes I can add `screencasting` to the dictionary. But I will that in a new PR, hence I am closing this one and open a new PR because after rebasing I am getting changes in other files also. So I will close this.

Reply:
sounds good, sorry this got stale